unity-renderer/Assets/DCLPlugins/FallbackAssetLoaderPlugin/FallbackFontsLoaderPlugin.cs

```diff
@@ -0,0 +1,34 @@
+using DCL;
+using System.Collections.Generic;
+using TMPro;
+
+namespace DCLPlugins.FallbackFontsLoader
+{
+    public class FallbackFontsLoaderPlugin : IPlugin
+    {
+        private readonly string FALLBACK_FONTS_ADDRESS = "Assets/Fonts/Fonts Sources/";
```

Comment:
change to `const` as it should not be an instance field

Reply:
Done!